samples/server/petstore/springboot-reactive-noResponseEntity/src/main/java/org/openapitools/api/PetApiDelegate.java

```diff
@@ -76,7 +76,7 @@ default Mono<Void> deletePet(Long petId,
      *         or Invalid status value (status code 400)
      * @see PetApi#findPetsByStatus
      */
-    default Mono<Flux<Pet>> findPetsByStatus(List<String> status,
+    default Flux<Pet> findPetsByStatus(List<String> status,
         ServerWebExchange exchange) {
         Mono<Void> result = Mono.empty();
```

Comment:
Should this not be Flux and Flux#empty

Reply:
Yes, ideally it should. But that would require a lot more changes for what is a "default implementation" that user should override.
In particular, that would require:
* either generate a list of objects instead of strings in JSON/XML
* or parse the JSON/XML in ApiUtil (convert the string to objects) to return a Flux afterwards

This looks unnecessary to me.